deap/algorithms.py

```diff
@@ -226,22 +226,34 @@ def varOr(population, toolbox, lambda_, cxpb, mutpb):
         "The sum of the crossover and mutation probabilities must be smaller "
         "or equal to 1.0.")
 
+    # adjust probabilities since both crossover children are appended
+    cxpb_adj = cxpb / (2 - cxpb)
+    if cxpb != 1.0:         #  Avoid zero division
```

Comment:
Shouldn't it be cxpb_adj?

Reply:
In the if-clause both variants would work since cxpb_adj only ever becomes 1 if cxpb equals 1.
The possible zero divison this check stops from happening is in `mutpb / (mutpb + (1 - mutpb - cxpb))` and occurs when cxpb = 1.0 and mutpb = 0.
Thats why I thought it would be more clear to check cxpb directly.